pkg/ctl/utils/describe_stacks.go

```diff
@@ -71,6 +97,13 @@ func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool) error {
 		logger.Warning("only %d stacks found, for a ready-to-use cluster there should be at least 2", len(stacks))
 	}
 
+	if printer != nil && output == printers.JSONType {
+		return printers.NewJSONPrinter().PrintObj(stacks, os.Stdout)
+	}
+	if printer != nil && output == printers.YAMLType {
+		return printers.NewYAMLPrinter().PrintObj(stacks, os.Stdout)
+	}
```

Comment:
```suggestion
```suggestion
	if printer != nil {
		return printer.PrintObj(stacks, os.Stdout)
	}
```
```